### PR TITLE
VTCCA-2872: re populate user details from session store when user arr…

### PR DESCRIPTION
**Pull request by habeeb01**
…ives on the registration page

### CODE REVIEW

File: app/models/registration/OrganisationUserAccountDetails.scala [hunk 1]
```diff
@@ -22,39 +22,39 @@ import models.domain._
...
+//    isAgent = isAgent
+//  )
+//}
+//
+//object AdminOrganisationAccountDetails {
+//  implicit val format = Json.format[AdminOrganisationAccountDetails]
+//}
+//
+//case class GroupAccountDetails(companyName: String, address: Address, email: String, confirmedEmail: String,
+//                               phone: String, isAgent: Boolean)
```
Comment (darrell-plant) [inline app/models/registration/OrganisationUserAccountDetails.scala]: commented-out code

File: app/models/registration/User.scala [hunk 2]
```diff
@@ -18,38 +18,28 @@ package models.registration
...
 import form.Mappings._
+import form.TextMatching
 import models.domain._
+import models.{Address, IVDetails, IndividualAccountSubmission, IndividualDetails, email => _}
 import play.api.data.Forms._
 import play.api.data.validation._
 import play.api.data.{Form, Mapping}
+import play.api.libs.json._
+import utils.EmailAddressValidation
+import views.helpers.Errors
```
Comment (darrell-plant) [inline app/models/registration/User.scala]: unused imports

**Comment by darrell-plant** [comment]
lgtm 👍

File: app/models/registration/FieldData.scala [hunk 3]
```diff
@@ -38,6 +43,50 @@ object FieldData {
       businessName = "",
       businessPhoneNumber = "",
       businessAddress = Address(None, "", "", "", "", ""),
-      isAgent = true
+      isAgent = true //fixme why is this true ???
```
Comment (jon-pike) [inline app/models/registration/FieldData.scala]: Another fixme

File: app/models/registration/OrganisationUserAccountDetails.scala (hunk 1 above)
Comment (habeeb01) [reply app/models/registration/OrganisationUserAccountDetails.scala]: file is now deleted

File: app/models/registration/User.scala (hunk 2 above)
Comment (habeeb01) [reply app/models/registration/User.scala]: imports removed

File: app/models/registration/FieldData.scala (hunk 3 above)
Comment (habeeb01) [reply app/models/registration/FieldData.scala]: fixme removed

**Comment by jon-pike** [comment]
👍

**Comment by darrell-plant** [comment]
👍 👍

**Pull request merged**